packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/comfort-exchange.html

```diff
@@ -74,22 +74,42 @@
                     data-translate="xdsl_access_comfort_exchange_warning_part2"
                 ></p>
             </oui-message>
-            <div
-                class="mb-2"
-                data-translate="xdsl_access_comfort_exchange_confirm_title"
-            ></div>
-            <div
-                class="mb-2"
-                data-translate="xdsl_access_comfort_exchange_confirm_subtitle"
-            ></div>
+            <!-- CONTACT CHOICE -->
+            <div class="contact-choice col-xs-12 col-md-6">
```

Comment:
Since you're using `col-*` helpers, you must rely on parent `row` element.
see: https://github.com/twbs/bootlint/wiki/E014